recipes-connectivity/networkmanager/networkmanager/99-disable-uap.conf

```diff
@@ -0,0 +1,5 @@
+[main]
+plugins=keyfile
+
+[keyfile]
+unmanaged-devices=interface-name:uap*
```

Comment:
Nit-pick: missing newline.

Reply:
done! also updated the commits on Scarthgap and Master.